### PR TITLE
Add exact matching section

### PR DESCRIPTION
**Pull request by NohYujeong**

### CODE REVIEW

File: app/api/search.ts [hunk 1]
```diff
@@ -49,6 +49,7 @@ export interface SearchResult extends PaginationResponseV2<Paper[]> {
     resultModified: boolean;
     suggestion: Suggestion | null;
     detectedYear: number | null;
+    detectedPhrases: string[];
```
Comment (TylorShin) [inline app/api/search.ts]: is it nullable?

File: app/components/articleSearch/components/PaperSearchResultInfo/index.tsx [hunk 2]
```diff
@@ -32,6 +34,25 @@ const PaperSearchResultInfo: React.FC<PaperSearchResultInfoProps> = ({
     </span>
   );
 
+  if (matchingPhrases && matchingPhrases.length > 0) {
+    const clearPhrases = matchingPhrases.map((phrases, i) => {
+      if (i < matchingPhrases.length - 1) return `"${phrases}", `;
+      else return `"${phrases}"`;
+    });
```
Comment (TylorShin) [inline app/components/articleSearch/components/PaperSearchResultInfo/index.tsx]:
```
const phrasesString = matchingPhrases.map(word => `"${word}"`).join(', ');
```
will be short and easy to understand.

File: app/api/search.ts (hunk 1 above)
Comment (NohYujeong) [reply app/api/search.ts]: No, it always an empty array when there is no item.

File: app/components/articleSearch/components/PaperSearchResultInfo/index.tsx (hunk 2 above)
Comment (NohYujeong) [reply app/components/articleSearch/components/PaperSearchResultInfo/index.tsx]: 👍

**Pull request merged**